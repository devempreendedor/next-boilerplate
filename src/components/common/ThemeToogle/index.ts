export { default as ThemeToogle } from './ThemeToogle'
