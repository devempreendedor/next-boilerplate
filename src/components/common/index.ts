export * from './ThemeToogle'
export * from './Layout'
