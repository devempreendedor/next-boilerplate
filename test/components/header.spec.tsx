import { render, screen } from '../test-utils'
import { Header } from '@components/common'

describe('Header Component', () => {
  it('Should render the title on Header', () => {
    render(<Header />)

    const title = screen.getByText(/Boiler/i)

    expect(title).toBeInTheDocument()
    expect(title).toMatchSnapshot()
  })
})
