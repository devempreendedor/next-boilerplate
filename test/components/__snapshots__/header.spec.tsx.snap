// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Should render the title on Header 1`] = `
<div>
  Boiler
</div>
`;
